src/main/cpp/native_glue.hh

```diff
@@ -75,17 +71,15 @@ namespace litecore {
 
             jbyteArraySlice(jbyteArraySlice &&s) // move constructor
                     : _slice(s._slice), _env(s._env), _jbytes(s._jbytes),
-                      _critical(s._critical) { s._slice = nullslice; }
-
-            operator slice() { return _slice; }
+                      _critical(s._critical) { s._slice = kFLSliceNull; }
 
-            operator C4Slice() { return {_slice.buf, _slice.size}; }
+            operator FLSlice() { return _slice; }
 
             // Copies a Java byte[] to an alloc_slice
-            static alloc_slice copy(JNIEnv *env, jbyteArray jbytes);
+            static FLSliceResult copy(JNIEnv *env, jbyteArray jbytes);
```

Comment:
// Copies a Java byte[] to an " `FLSliceResult`  "

Reply:
Will fix this. Thanks.